fill/proto/interactive_filler.go

```diff
@@ -254,13 +254,13 @@ func (r *resolver) resolveField(f *desc.FieldDescriptor) error {
 	}
 }
 
-func (r *resolver) resolveEnum(e *desc.EnumDescriptor) (int32, error) {
+func (r *resolver) resolveEnum(prefix string, e *desc.EnumDescriptor) (int32, error) {
 	choices := make([]string, 0, len(e.GetValues()))
 	for _, v := range e.GetValues() {
 		choices = append(choices, v.GetName())
 	}
 
-	choice, err := r.selectChoices(e.GetFullyQualifiedName(), choices)
+	choice, err := r.selectChoices(prefix, choices)
```

Comment:
Since the prompt has an unnecessary character `:`, it's better to trim it like `? choice (TYPE_ENUM) =>`.
This can be pass a template through `Templates.Label` of [`promptui.Select`](https://pkg.go.dev/github.com/manifoldco/promptui#Select) (the implementation of the prompt).

FYI: the default template label is maybe [here](https://github.com/manifoldco/promptui/blob/c2e487d3597f59bcf76b24c9e80679740a72212b/select.go#L421).

![image](https://user-images.githubusercontent.com/12953836/148952269-03bb91e0-70aa-4c9e-b1c4-51bc8d4fd313.png)

Reply:
I updated the diff.